src/utils/searchUtils.ts

```diff
@@ -50,7 +51,7 @@ export const filterScenarios = (
         scenario.modelTempIncrease,
         ...scenario.regions,
         ...scenario.sectors.map((s) => s.name),
-        ...scenario.sectors.map((s) => s.tooltip || ""),
+        ...scenario.sectors.map((s) => getSectorTooltip(s.name) || ""),
```

Comment:
@jdhoffa @lloyd-rmi Currently the sector tooltip text is included in the search corpus, but not pathway type tooltip. 

it feels wewird to have one, but not the other. Given that these are static strings now, I don't think it makes sense to keep them searchable, but your call.

Reply:
Flagged as #256, suggest we sort it out there, either way